build-support/travis/travis.yml.mustache

```diff
@@ -377,17 +377,10 @@ py27_linux_build_wheels_ucs2: &py27_linux_build_wheels_ucs2
   env:
     - *base_build_wheels_env
     - docker_image_name=travis_ci_py27_ucs2
+    - docker_run_command="./build-support/bin/ci.sh -2b
+                          && ./build-support/bin/check_pants_pex_abi.py cp27m
+                          && RUN_PANTS_FROM_PEX=1 ./build-support/bin/release.sh -n"
```

Comment:
Having a comment somewhere on how this gets injected into the script that gets run would be neat, since there's a bit of indirection where `${docker_run_command}` is used in the expansion of `{{>docker_run_image}}`.